pkg/helm/helm_test.go

```diff
@@ -1,10 +1,29 @@
+// Copyright 2023 Greptime Team
```

Comment:
Note: The year is 2023 here. In `pkg/deployer/k8s/deployer.go` and `pkg/deployer/types.go` is 2022.

Reply:
@daviderli614 we can use `license-eye` to update all the copyright in next PR.